### PR TITLE
Swap to use the new GitHub releases since irssi.org no longer hosts the tarballs

### PR DESCRIPTION
**Pull request by tianon**
See https://irssi.org/2015/12/14/irssi-website-now-on-github-pages/ for a little bit more context.

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eo pipefail
 
-current="$(curl -fsSL 'http://www.irssi.org/download' | grep '<li>Latest release version: ' | sed -r 's!^.*<li>Latest release version: <strong>([^"]+)</strong></li>.*$!\1!' | head -1)"
+current="$(git ls-remote --tags https://github.com/irssi-import/irssi.git | cut -d/ -f3 | cut -d^ -f1 | sort -uV | grep -v -- -rc | tail -1)"
```
Comment (tianon) [inline update.sh:4]: On the plus side, no more HTML scraping, and no more failing builds when irssi.org goes down. :smile:

Comment (jessfraz) [inline update.sh:4]: nice!!!

On Fri, Dec 18, 2015 at 11:50 PM, Tianon Gravi notifications@github.com
wrote:

> In update.sh
> https://github.com/jfrazelle/irssi/pull/3#discussion_r48090089:
> 
> > @@ -1,7 +1,7 @@
> >  #!/bin/bash
> >  set -eo pipefail
> > 
> > -current="$(curl -fsSL 'http://www.irssi.org/download' | grep '<li>Latest release version: ' | sed -r 's!^._<li>Latest release version: <strong>([^"]+)</strong></li>._$!\1!' | head -1)"
> > +current="$(git ls-remote --tags https://github.com/irssi-import/irssi.git | cut -d/ -f3 | cut -d^ -f1 | sort -uV | grep -v -- -rc | tail -1)"
> 
> On the plus side, no more HTML scraping, and no more failing builds when
> irssi.org goes down. [image: :smile:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jfrazelle/irssi/pull/3/files#r48090089.

## 

Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3

**Comment by tianon** [comment]
https://hub.docker.com/r/armhf/irssi/tags/ :smiling_imp:

**Comment by jessfraz** [comment]
Niceeeeee

**Pull request merged**